corehq/motech/fhir/data/capability_statement.json

```diff
@@ -0,0 +1,41 @@
+{
+    "rest": [
+        {
+            "security": {
+                "extension": [
+                    {
+                        "url": "http://fhir-registry.smarthealthit.org/StructureDefinition/oauth-uris",
+                        "extension": [
+                            {
+                                "url": "token",
+                                "valueUri": "__TOKEN_URL__"
+                            },
+                            {
+                                "url": "authorize",
+                                "valueUri": "__AUTHORIZE_URL__"
+                            }
+                        ]
+                    }
+                ],
+                "service": [
+                    {
+                        "coding": [
+                            {
+                                "system": "http://hl7.org/fhir/restful-security-service",
+                                "code": "SMART-on-FHIR"
+                            }
+                        ]
+                    }
+                ]
+            },
+            "mode": "server"
+        }
+    ],
+    "format": [
+        "json"
+    ],
+    "status": "active",
+    "kind": "instance",
+    "fhirVersion": "__FHIR_VERSION__",
+    "date": "__PUBLISHED_DATE__"
+}
```

Comment:
omg yes! Thank you!

Every time we statically define something that could just be a dictionary, a Python fairy dies, and a Java demon does that "MWHAHAHAHA" laugh from the pit of Python fairy hell, knowing that every time a developer needs to do anything with what could just have been a dictionary, they first have to negotiate with nested classes.

Reply:
Yeah, I don't know what I was thinking.

Reply:
Oh, as you know, there is a lot of precedent in our codebase, and I've done the same myself. (In a pre-Dimagi life I spent a year building a platform that (among other things) inspected remote APIs and used Python metaclasses to generate classes to represent API documents/resources. Way down the line I realized that I could have avoided all the metaclass work by just using dictionaries, and, if they needed to be validated, checking them against a generated schema -- i.e. what we are doing with FHIR resources.)